rasp/selenium_engine.py

```diff
@@ -1,91 +1,31 @@
 import selenium
 import selenium.webdriver
-import time
 
 from rasp.base import Engine, Webpage
 from rasp.errors import EngineError
 
 
 class BaseSeleniumEngine(Engine):
-    def __init__(self):
-        self.driver = None
-        return
+    def __init__(self, *args, **kwargs):
+        self.driver = selenium.webdriver.Firefox()
```

Comment:
how do we feel about this change?  it seemed to me like the `setup` method would be better used as a hook for custom behavior.  that said, the cleanup method is currently also not used for custom behavior.  we could use engines as a contextmanager and take care of setting up and tearing down automatically.